packages/sources/polygon/src/endpoint/tickers.ts

```diff
@@ -0,0 +1,78 @@
+import { Requester, Validator } from '@chainlink/ea-bootstrap'
+//import { warmupStopped } from '@chainlink/ea-bootstrap/src/lib/cache-warmer/actions'
+import { ExecuteWithConfig, Config, InputParameters, AdapterRequest, AxiosResponse } from '@chainlink/types'
+import { NAME as AdapterName } from '../config'
+
+export const supportedEndpoints = ['tickers']
+export const batchablePropertyPath = ['base', 'quote']
+
+export const inputParameters: InputParameters = {
+  base: ['base', 'from'],
+  quote: ['quote', 'to'],
+  quantity: false,
+}
+
+const handleBatchedRequest = (
+  jobRunID: string,
+  request: AdapterRequest,
+  response: AxiosResponse<any>,
+  resultPath: string[],
+) => {
+  const payload: [AdapterRequest, number][] = []
+  for (const pair of response.data.tickers) {
+    const base = pair.ticker.substring(2,5)
```

Comment:
Does this handle tickers that are more than `3` characters?

Reply:
No it wouldn't, I didn't know that there could be ones that are longer than 3 characters...